imagePNG.ml

```diff
@@ -881,19 +888,20 @@ module ReadPNG : ReadImage = struct
          (* Removing the filter on the scanlines *)
          let prev_scanline = ref None in
          Array.mapi
-           (fun y (ftype, scanline) ->
+           (fun _y (ftype, scanline) ->
```

Comment:
Why not use `_`? Or even better, use `Array.map` instead of `Array.mapi` since we do not use the index.

Reply:
I think `Array.map` would be better here, wasn't sure if you wanted to keep the `y` for future reference. I'll change it to that!